src/v/model/fundamental.h

```diff
@@ -123,6 +123,54 @@ class topic : public named_type<ss::sstring, struct model_topic_type> {
 using ns = named_type<ss::sstring, struct model_ns_type>;
 
 using offset = named_type<int64_t, struct model_offset_type>;
+using offset_delta = named_type<int64_t, struct model_offset_delta_type>;
```

Comment:
nit: this might benefit from a comment about the context of "delta"

I was initially confused by this: I thought "surely adding a delta to a kafka offset should give me another kafka offset?" until I realised this is specifically deltas _for offset translation_ rather than deltas between two offsets in the same space.

Reply:
added a doc-comment